test/sass/logical-properties.spec.scss

```diff
@@ -0,0 +1,980 @@
+@use "test" as *;
+@use "../../src/shared-styles/validation" with ( $is-test: true);
+@use "../../src/shared-styles/logical-properties";
+
+/* stylelint-disable csstools/use-logical  */
+
+@include describe("@mixin logical-property") {
+
+  @include it("generates correct fallbacks for the dimension 'inline'") {
+
+    @include assert() {
+
+      @include output {
+        @include logical-properties.property(margin, inline, 16px);
```

Comment:
Hmm, wonder if there's a better name for either/both the namespace and mixin.

Reply:
I'm not happy with the naming, but haven't come up with anything better yet. Suggestions welcome!

Reply:
Update: see https://github.com/libero/storybook/pull/34#discussion_r345675983